olm-catalog/serverless-operator/csv.template.yaml

```diff
@@ -273,7 +273,7 @@ spec:
                       fieldPath: metadata.namespace
                 - name: METRICS_DOMAIN
                   value: knative.dev/serving-operator
-                image: registry.svc.ci.openshift.org/openshift/knative-v0.17.0:knative-operator
+                image: registry.svc.ci.openshift.org/openshift/openshift-serverless-nightly:openshift-knative-operator
```

Comment:
we promote the image on `openshift/release` already?

Reply:
Yepo!